src/Home.jsx

```diff
@@ -13,10 +14,10 @@ const Home = () => {
             </p>
           </header>
 
-          <div className="usa-alert usa-alert-info">
```

Comment:
Why the switch from the uswds classes here? Aren't the base definitions still used for the unspecified css?